build-system/common/ci.js

```diff
@@ -180,7 +180,7 @@ function ciBuildUrl() {
     : isGithubActions
     ? `${env('GITHUB_SERVER_URL')}/${env('GITHUB_REPOSITORY')}/actions/runs/${env('GITHUB_RUN_ID')}` // prettier-ignore
     : isCircleci
-    ? env('CIRCLE_BUILD_URL')
+    ? `https://app.circleci.com/pipelines/workflows/${env('CIRCLE_WORKFLOW_ID')}` // prettier-ignore
```

Comment:
No way to avoid linter-overrides? :'(

Reply:
Not without making it span multiple lines. I'll come back and clean up this file once I research ways to extract the entire URL from other env vars.